### PR TITLE
Add a biter zombie

### PR DESCRIPTION
**Pull request by jokermatt999**
Fairly tough, likes to bite a lot. Considering giving him a special with more infectiousness, but for now I figure biting more should be enough.

### CODE REVIEW

**Comment by Coolthulhu** [comment]
Would make sense as a decayed zed evolution, those bite a lot too.

**Comment by jokermatt999** [comment]
I'm thinking of having a venomous upgrade to this with an upgraded
grab/constrict and poisonous bite, which is why I went this direction.
Should've mentioned that in the PR, but I wanted to see what people thought
of this one first.

**Comment by chaosvolt** [comment]
Hmm. Seems interesting, and venom would be interesting as an upgrade.

**Comment by jokermatt999** [comment]
Ok, fixed the json, and did a little bit of testing. You can take them down with a new evacuee survivor with 10 stats and 0 skill, but you'll almost certainly get infected on at least 1 body part, which is right where this should be. I think I'll avoid buffing the grab to give players a chance to get away if they get too close.

**Comment by Lain-** [comment]
Seems nice, having this zombie be an evolution of the decayed zombie makes sense as right now decayed zombie doesn't do anything too interesting.

**Comment by jokermatt999** [comment]
Decayed zombies don't really fit for the evolution I'm going to have after this. It's going to be a beefier grabber with crocodile like jaws and a toxic bite. Decayed zombies are losing mass, not gaining it. I do have some ideas for them too though.

File: data/json/monsters.json [hunk 1]
```diff
@@ -2580,6 +2580,35 @@
...
+    "dodge":1,
+    "armor_bash":5,
+    "armor_cut":1,
+    "vision_day":15,
+    "vision_night":4,
+    "death_drops": "default_zombie_death_drops",
+    "hp":100,
+    "death_function":["NORMAL"],
+    "special_attacks":[["BITE", 2], ["GRAB", 4]],
+    "description":"A distorted and swollen human body. Its jaws have elongated into a crocodile like snout, dripping with foul smelling saliva. Worse yet, the arms are bulging with clusters of sinewy muscle.",
```
Comment (Coolthulhu) [inline data/json/monsters.json]: Should have 2 spaces after sentence end (except last sentence, obviously)

**Comment by Coolthulhu** [comment]
They're quite interesting in that they have crap senses but good speed. This makes them pop up from the darkness, while also making them easy to avoid in daylight.

Their evolution could have a charge attack to facilitate this sudden appearance.

**Comment by chaosvolt** [comment]
Ooh, interesting. X3

**Comment by jokermatt999** [comment]
@Coolthulhu :) Glad you saw where I was going with that. It's going to have a charge (looking at the irradiated wanderers attack to base it off of), and then a strong mauling attack on downed targets.

**Comment by Coolthulhu** [comment]
Mauling isn't really needed. Lying targets are fucked enough as is (they lose all the turns and get surrounded).

If you want to add a poison bite, it would be a good idea to bring decayed zed into it. Decayed has no evolution, grappler already has one.

**Comment by jokermatt999** [comment]
K, I'll hold off on the maul then. As for decayed zombies, I've got plans
for them. Think "blob tried to regenerate, goes all The Thing."

**Comment by chaosvolt** [comment]
Interesting. Already working on adding a sprite for it. o3o

**Pull request merged**